src/cci/cci_util.c

```diff
@@ -772,6 +775,82 @@ ut_double_to_str (double value, char *str, int size)
   snprintf (str, size, "%.16f", value);
 }
 
+void
+ut_double_to_trailingzero_str (double value, char *str, int size)
```

Comment:
Propose function name as:

ut_double_to_str_with_remove_trailingzeros ()

Reply:
thanks for your comment